src/components/projects/projectsList.js

```diff
@@ -0,0 +1,64 @@
+// import { FaDiscord, FaEarthAmericas, FaGithub } from 'react-icons/fa6';
+
+const projectsList = [
+  {
+    title: 'RecordSponge',
+    description:
+      "RecordSponge is an app that helps community organizations quickly analyze an individual's criminal history to determine if they qualify to have their records expunged.",
+    status: 'Live',
+    logo: { image: '/assets/logoRecordSponge.svg', alt: 'RecordSponge logo' },
+    links: [
+      {
+        href: 'https://github.com/codeforpdx/recordexpungPDX',
+        // icon: 'FaGithub',
+        // icon: <FaGithub size={45} />,
+        icon: '<FaGithub size={45} />',
+        ariaLabel: 'Github link'
```

Comment:
ariaLabel in here can all be removed (they arent being used at least in projectsBreif.jsx) and added as a template literal like the alt text.